packages/completer/src/widget.ts

```diff
@@ -126,6 +128,10 @@ export class Completer extends Widget {
     }
   }
 
+  set showDocsPanel(showDoc: boolean) {
+    this._showDoc = showDoc ?? true;
```

Comment:
Either showDoc could be undefined (the signature needs to be corrected) or the nullish operator is not needed.

```suggestion
    this._showDoc = showDoc ?? true;
```